packages/deployments/contracts/contracts/core/connext/facets/TokenFacet.sol

```diff
@@ -418,7 +449,10 @@ contract TokenFacet is BaseConnextFacet {
 
     // Delete from canonical mapping
     delete s.canonicalToAdopted[_key];
-    delete s.canonicalToRepresentation[_key];
+
+    // NOTE: we do NOT delete the entry from the `canonicalToRepresentation`
+    // mapping. This is done to prevent multiple representations being deployed
+    // in `setupAsset`
```

Comment:
Note, since we are not deleteing this value. The owner or an admin can still update a removed asset's old representation token:

https://github.com/connext/nxtp/blob/d79e6e7c43123ee172af5cd44dcc3c80ab476f96/packages/deployments/contracts/contracts/core/connext/facets/TokenFacet.sol#L296-L307

Reply:
fixed:
- `updatedDetails()`

see [here](https://github.com/spearbit-audits/connext-nxtp/issues/80#issuecomment-1319292417) for more details on fixes not applied